src/index.tsx

```diff
@@ -1,90 +1,89 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import {
-  createBrowserRouter,
-  RouterProvider,
-} from "react-router-dom";
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import './index.css';
 import reportWebVitals from './reportWebVitals';
 import './register-webcomponents';
 
-import App from "./App";
-import WebComponents from "./pages/WebComponents";
-import TextPage from "./pages/TextPage/TextPage";
+import App from './App';
+import WebComponents from './pages/WebComponents';
+import TextPage from './pages/TextPage/TextPage';
 import TablePage from './pages/TablePage';
 import FormPage from './pages/FormPage';
 import TabsLinkPage from './pages/TabsLinkPage';
 import TabsButtonPage from './pages/TabsButtonPage';
 import About from './pages/About';
 import SettingsPage from './pages/SettingsPage';
-
+import TextSubpage from './pages/TextSubpage';
 
 const router = createBrowserRouter([
   {
-    path: "/",
+    path: '/',
     element: <App />,
     children: [
       {
-        path: "",
+        path: '',
         element: <WebComponents />,
       },
       {
-        path: "web-components",
+        path: 'web-components',
         element: <WebComponents />,
       },
       {
-        path: "table",
+        path: 'table',
```

Comment:
Not a part of this PR, but looks like we lost a navigation item for the table page, right?

Reply:
I think it's the one just below this comment, right?

Reply:
I should be more specific: I meant in Header or Side menu. Route works as intended if you type it, but we have no navigation item in Header or Side menu to it.

Reply:
Aah alright!

Reply:
Yeah it looks like the side-menu is a bit off on mobile also. The banner is covering the close button. Maybe fix it in another PR?

Reply:
Yep, another PR for sure, just notice it when going through this one  ;)